slurm-script/kb_init.sh

```diff
@@ -10,4 +9,4 @@ set -e # Abort if any command fails
 
 export JAVA_OPTS="-Xmx70G"
 
-make all
\ No newline at end of file
+make -f ../Makefile all
```

Comment:
Won't this confuse `make` in finding the targets? I thought you would run from the root directory where Makefile is: `./slurm-script/kb_init.sh`, and that would just contain `make all`? Am I thinking about it wrong?